### PR TITLE
reDesign v2

### PR DESCRIPTION
**Pull request by simonweniger**
This one fixes the issues: #5  #10  #12  #6  #7  #8  #11 #9 #14 

- New Navbar
- Fixed wrong home layout
- small style changes
- some renaming from moloch to rosebud
-cleaned the assets folder
- changed from ETH to DAI (WIP)
- new submission form design
- fixed Dropdown Menu
- deleted logged in users display
- improved member view layout and design
- improved proposal list view
- added the new proposal form to the proposal list tab for a smoother user experiance
- first improvments of proposal detail view (WIP)
- proposal card redesign (page redesign)
- fix dai to dollar view
- re designed proposal detail view
- display 3box names

### CODE REVIEW

File: packages/frontend/src/components/MemberList.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React from "react";
-import { Grid, Image, Divider } from "semantic-ui-react";
+import { Grid, Image,Segment } from "semantic-ui-react";
```
Comment (johba37) [inline packages/frontend/src/components/MemberList.js:2]:
```suggestion
import { Grid, Image, Segment } from "semantic-ui-react";
```

File: packages/frontend/src/components/ProposalDetail.js [hunk 2]
```diff
@@ -201,29 +229,31 @@ const ProposalDetail = ({ loggedInUser, match }) => {
...
                   <Grid>
                     <Grid.Row className="members_row">
                       {/* centered */}
                       {proposal.votes.map((vote, idx) => (
-                        <MemberAvatar
-                          member={vote.member.id}
-                          shares={vote.member.shares}
-                          key={idx}
+                        <ProfileHover
+                        address={vote.member.id}
```
Comment (johba37) [inline packages/frontend/src/components/ProposalDetail.js]: indentation missing

File: packages/frontend/src/components/ProposalDetail.js [hunk 3]
```diff
@@ -275,8 +311,9 @@ const ProposalDetail = ({ loggedInUser, match }) => {
               </Grid>
             </Grid.Row>
           </Grid>
-        </Grid.Column>
+        </Grid.Column>
```
Comment (johba37) [inline packages/frontend/src/components/ProposalDetail.js]: no trailing spaces

File: packages/frontend/src/components/ProposalSubmission.js [hunk 4]
```diff
@@ -11,7 +10,7 @@ import {
   Icon,
   List,
 } from "semantic-ui-react";
-import { getMoloch, getToken } from "../web3";
+import { getMoloch, getToken, } from "../web3";
```
Comment (johba37) [inline packages/frontend/src/components/ProposalSubmission.js:13]:
```suggestion
import { getMoloch, getToken } from "../web3";
```

File: packages/frontend/src/components/ProposalSubmission.js [hunk 5]
```diff
@@ -41,6 +40,9 @@ class SubmitModal extends Component {
       beneficiaryApproved = true;
     }
 
+    console.log(loggedInUser);
```
Comment (johba37) [inline packages/frontend/src/components/ProposalSubmission.js]: no console log

File: packages/frontend/src/components/ProposalSubmission.js [hunk 6]
```diff
@@ -64,17 +66,18 @@ class SubmitModal extends Component {
...
     const { handleSubmit, submittedTx } = this.props;
     return (
-      <Modal
+    <div id="proposal_submission">
+      <Modal 
         trigger={
-          <Button size="large" color="red" onClick={this.handleOpen}>
-            Submit Proposal
+          <Button size="large" color="green" onClick={this.handleOpen} >
+            <Icon name='send' color="white"/>  Submit a new proposal
```
Comment (johba37) [inline packages/frontend/src/components/ProposalSubmission.js:70]:
```suggestion
            <Icon name="send" color="white"/>  Submit a new proposal 
```

File: packages/frontend/src/App.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import { useQuery } from "@apollo/react-hooks";
 import { HttpLink, ApolloClient, InMemoryCache } from "apollo-boost";
 import gql from "graphql-tag";
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect} from "react";
```
Comment (johba37) [inline packages/frontend/src/App.js:4]:
```suggestion
import React, { useState, useEffect } from "react";
```

File: packages/frontend/src/components/Header.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useCallback, useEffect } from "react";
 import { Link } from "react-router-dom";
-import { Grid, Icon, Dropdown, Form, Button, Loader, Menu } from "semantic-ui-react";
+import { Icon, Dropdown, Form, Button, Loader,Menu} from "semantic-ui-react";
```
Comment (johba37) [inline packages/frontend/src/components/Header.js:3]:
```suggestion
import { Icon, Dropdown, Form, Button, Loader, Menu } from "semantic-ui-react";
```

File: packages/frontend/src/components/ProposalDetail.js [hunk 9]
```diff
@@ -232,40 +262,46 @@ const ProposalDetail = ({ loggedInUser, match }) => {
...
-                    onClick={() => handleYes(proposal)}
-                  >
-                    Vote Yes
-                  </Button>
+                <Grid.Column textAlign="left">
+                  <ButtonGroup>
+                    <Button
+                      className="btn"
+                      color="green"
+                      size="mediun"
```
Comment (johba37) [inline packages/frontend/src/components/ProposalDetail.js:282]:
```suggestion
                      size="medium"
```

**Review by johba37 — APPROVED** [review]
great work 👏 

what i've done in review:
- [x] ran project locally and tested general flow
- [x] reviewed code and pointed out syntax issues
- [x] tested each of the addressed issues locally

File: packages/frontend/src/components/ProposalDetail.js (hunk 2 above)
Comment (simonweniger) [reply packages/frontend/src/components/ProposalDetail.js]: solved

**Review by simonweniger — COMMENTED** [review]
accepted the suggestions and changed it in the code

**Pull request merged**